Makefile

```diff
@@ -7,27 +7,39 @@ CONTAINER_ENGINE?=$(shell command -v podman 2>/dev/null || echo "docker")
 unexport REPO_NAME
 
 .PHONY: isclean
-isclean:
+isclean: ## Validate the local checkout is clean. Use to ALLOW_DIRTY_CHECKOUT=true to nullify
 	@(test "$(ALLOW_DIRTY_CHECKOUT)" != "false" || test 0 -eq $$(git status --porcelain | wc -l)) || (echo "Local git checkout is not clean, commit changes and try again." >&2 && exit 1)
 
 .PHONY: tag-check
-tag-check:
+tag-check: ## Perform a tag-check that validates a new tag has been created when changing the build image
 	config/tag-check.sh
 
 .PHONY: test
-test: isclean tag-check
+test: ## Runs tests under the /case directory
+	isclean tag-check
 	test/driver $(CASE_GLOB)
 
 .PHONY: pr-check
-pr-check: test
+pr-check: ## Performs `make test`
```

Comment:
FWIW, the reason this seemingly redundant target exists is to be the thing run by CI. At some point in the future we may add things to it other than `test`. Not asking for a change here, but if you felt like saying “this is the target run by prow” that would be cool.